com.zettlr.Zettlr.metainfo.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally, it should be submitted and maintained upstream.

Reply:
It really isn't useful for electron apps since their buildsystems don't support injecting custom metainfo files.